pkg/kubelet/eviction/helpers.go

```diff
@@ -714,8 +714,15 @@ func makeSignalObservations(summaryProvider stats.SummaryProvider, nodeProvider
 			}
 		}
 	}
-	if memoryAllocatableCapacity, ok := node.Status.Allocatable[v1.ResourceMemory]; ok {
+
+	nodeCapacity := containerManager.GetCapacity()
+	allocatableReservation := containerManager.GetNodeAllocatableReservation()
+
+	if memoryAllocatableCapacity, ok := nodeCapacity[v1.ResourceMemory]; ok {
```

Comment:
How about a helper method that takes in a resource name and spits out `Allocatable`?